docs/Guides/CustomerVoice/SendReviewRequest.md

```diff
@@ -0,0 +1,237 @@
+---
+tags: [Review Request, Customer Voice]
+---
+# Request a Review for a Business Location
+
+> This guide relies on "Trusted Tester" endpoints and is likely to change until all features have been made generally available.
+
+With Customer Voice, you can gather authentic reviews via email or text on the sites that matter most, to grow customer loyalty and boost sales.
+
+## Setup
+Create an access token with `reviews` and `customers` scopes following the [Authorization guide](../../Authorization/Authorization.md).
+
+Have an activated Customer Voice product and SMS add-on if you need to send SMS review requests. Ensure your email and SMS templates are configured appropriately and your preferred review sites are configured within the Customer Voice platform.
+
+## Step 1: Obtain a Template ID
+Templates control the text and layout of your review request and can be configured to dynamically populate fields from your business information, a customer record or your preferred review sites. 
+
+To obtain a template ID, navigate to the "Templates" tab and select "Edit Template" from the configuration menu of your chosen template. 
+![Templates list page](./images/edit-template.png)
+
+Scrolling to the bottom of the "Edit Template" page you will see a button for copying the template ID.
```

Comment:
Since this button is only enabled for select partners, I'm thinking maybe we should include a statement that states if they don't see the button they should contact us to have the button enabled.

Reply:
Yep, I can do that. Is there any harm in just enabling this feature for everyone?

Reply:
Probably not.  I guess the only thing I can think of is it might add some confusion to people on why this button exists or why it showed up all of a sudden if they're not using the SFPT workflow or the APIs.  Also maybe a minor risk of a partner not wanting to have that button show up without being notified first.